core/com.b2international.snowowl.core.rest/src/com/b2international/snowowl/core/rest/SnowOwlApiConfig.java

```diff
@@ -214,18 +205,20 @@ protected void addFormatters(FormatterRegistry registry) {
 	}
 	
 	@Bean
-	public MethodValidationPostProcessor methodValidationPostProcessor() {
+	public static MethodValidationPostProcessor methodValidationPostProcessor() {
```

Comment:
Does the introduction of this `static` modifier have any significance? Are you calling it outside Spring's DI?

Reply:
No, we do not call this outside Spring's DI. Spring while boots prints this warning message to the console if this method is not static.
```
[2024-02-09T13:20:14.299] WARN  main  s.c.s.PostProcessorRegistrationDelegate$BeanPostProcessorChecker Bean 'snowOwlApiConfig' of type [com.b2international.snowowl.core.rest.SnowOwlApiConfig$$SpringCGLIB$$0] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying). The currently created BeanPostProcessor [methodValidationPostProcessor] is declared through a non-static factory method on that class; consider declaring it as static instead. 
```